modules/kubernetes/vpa/templates/vpa-values.yaml.tpl

```diff
@@ -0,0 +1,36 @@
+priorityClassName: platform-medium
+
+recommender:
+  # recommender.enabled -- If true, the vpa recommender component will be installed.
+  enabled: true
+  # recommender.extraArgs -- A set of key-value flags to be passed to the recommender
+  extraArgs:
+    v: "4"
+    pod-recommendation-min-cpu-millicores: 15
+    pod-recommendation-min-memory-mb: 24
+    prometheus-address: http://prometheus-operated.prometheus.svc.cluster.local:9090
+    storage: prometheus
+    # How much time back prometheus have to be queried to get historical metrics
+    # history-length: 8
```

Comment:
Got `Could not initialize history provider: history length 8 is not a valid Prometheus duration: not a valid duration string: 8`
Probably need to define if it should be hours days or something like that. Need to read the docs. 8 days is the default value